dace/codegen/targets/framecode.py

```diff
@@ -52,7 +51,8 @@ def generate_constants(self, sdfg: SDFG, callsite_stream: CodeIOStream):
                 callsite_stream.write(const_str, sdfg)
             else:
                 callsite_stream.write(
-                    "constexpr auto %s = %s;\n" % (cstname, str(cstval)), sdfg)
+                    "constexpr %s %s = %s;\n" % (csttype.dtype.ctype, cstname,
```

Comment:
This is great, now we can remove `generate_constants` from the FPGA codegen and just call this instead :-)

Reply:
:)